pkg/provider/aws.go

```diff
@@ -60,3 +61,26 @@ func (p AWSProvider) GetParameterValue(name string, decrypt bool) (string, error
 
 	return *param.Parameter.Value, nil
 }
+
+func (p AWSProvider) GetParameterDataByPath(path string, decrypt bool) (map[string]string, error) {
+	// path is something like /path/to/env
+	params, err := p.Service.GetParametersByPath(&ssm.GetParametersByPathInput{
+		Path:           aws.String(path),
+		Recursive:      aws.Bool(true),
+		WithDecryption: aws.Bool(decrypt),
+	})
+
+	if err != nil {
+		log.Fatal("Failed to get params by path '%s': %s", path, err)
+	}
+
+	results := make(map[string]string)
+	// '/path/to/env/foo' -> 'foo': *pa.Value
+	for _, pa := range params.Parameters {
+		basename := strings.Replace(*pa.Name, path, "", -1)
+		log.Infof("Param %s -> %s = %s", *pa.Name, basename, *pa.Value)
```

Comment:
Perhaps it should not log the values of the parameters. They are secrets, after all...

Reply:
Yeah, I'll remove the debugging stuff before I merge. Thanks for looking out!